src/extractors/custom/deadline.com/index.js

```diff
@@ -0,0 +1,34 @@
+export const DeadlineComExtractor = {
+  domain: 'deadline.com',
+
+  title: {
+    selectors: ['title'],
```

Comment:
Let's fetch the title from the article/body content. This article is a good example where the title on the page:

> Donald Trump Advises Boeing To Rebrand Max 737, Tweeting “But What The Hell Do I Know?”; Twitter Answers

doesn't exactly match the one in the `<title>` tag (which has been shortened for SEO purposes):

> Donald Trump Advises Boeing, Tweeting “But What The Hell Do I Know?”; Twitter Answers – Deadline

In theory, Mercury Parser should be parsing and processing content as it displays on the page.

Reply:
Thank you.
I fixed title selector `h1`.
However, I could not commit because `h1` innerText contained `&nbsp;`.
I could not do it, I decided to cut "Twitter Answers" from test.